edm4eic.yaml

```diff
@@ -140,6 +140,8 @@ components:
       - edm4eic::Cov2f    directionError  // Error on the polar and azimuthal angles
       - float             pathlength      // Pathlength from the origin to this point
       - float             pathlengthError // Error on the pathlength
+      - uint64_t          surface         // Surface track was propagated to (possibly multiple per detector)
+      - uint16_t          system          // Detector system track was propagated to
```

Comment:
I would like to see the following small changes:
- Make `system` a 32-bit integer as we aren't using 16-bit variables anywhere else. I was fine with use in pairs, but as a single value, use `uint32_t`
- Move `surface` and `system` to the top of the list (so the struct starts with the 64-bit variable). This makes it more consistent with our use of integer identifiers in other places and puts the large 64-bit integer before the 32-bit integers, so we don't have to worry about alignment.

Besides these small changes, I see no showstoppers.